app/services/create_fund_activity.rb

```diff
@@ -21,9 +22,13 @@ def call
     activity.accountable_organisation_reference = "GB-GOV-13"
     activity.accountable_organisation_type = "10"
 
-    activity.extending_organisation = Organisation.find_by!(service_owner: true)
```

Comment:
@lozette I would ❤️ to see why we wanted this in the private method in the commit message - I could learn so much about style.

Reply:
@tahb requested similar for projects here https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/303#discussion_r401512422 and I decided to go ahead and do it for funds as well 😆  Not sure if it was the right thing to do.